pretrain_language_model.py

```diff
@@ -101,8 +106,8 @@ def main():
     trainer.save_model()  # Saves the tokenizer too for easy upload
     metrics = train_result.metrics
 
-    max_train_samples = len(dataset)
-    metrics["train_samples"] = min(max_train_samples, len(dataset))
+    max_train_samples = len(train_dataset)
+    metrics["train_samples"] = min(max_train_samples, len(train_dataset))
```

Comment:
하나의 코드로 합쳐도 될 것 같습니다

```suggestion
    metrics["train_samples"] = len(train_dataset)
```